zulipterminal/helper.py

```diff
@@ -732,3 +734,52 @@ def suppress_output() -> Iterator[None]:
     finally:
         os.dup2(stdout, 1)
         os.dup2(stderr, 2)
+
+
+@asynch
+def process_media(controller: Any, media_link: str) -> None:
+    media_name = media_link.split("/")[-1]
+    client = controller.client
+    auth = requests.auth.HTTPBasicAuth(client.email, client.api_key)
+
+    with requests.get(media_link, auth=auth, stream=True) as r:
+        r.raise_for_status()
+        with NamedTemporaryFile(
+            mode="wb", delete=False, prefix="zt-", suffix="-{}".format(media_name)
+        ) as f:
+            media_path = f.name
+            for chunk in r.iter_content(chunk_size=8192):
+                if chunk:  # Filter out keep-alive new chunks.
+                    f.write(chunk)
+
+    if WSL:
+        command = f'explorer.exe `wslpath -w "{media_path}"`'  # Needs testing.
```

Comment:
Let's leave this out until we know we have a version that works, and we're sure is good, particularly with that quoting.

While we support only these three, we should handle the `else` case (or equivalent) in any case, so we'll need `command` to be defined, or have some meaning, and perhaps pop up an 'unsupported' notice or report an error. That means it can apply to the WSL case for now too.